lib/nodejs/manifest.js

```diff
@@ -181,12 +201,21 @@ function instances( applications ) {
           sessionKey = undefined;
         }
 
+        var participantCounts = {
+          instance: {
```

Comment:
Maybe the creation of this object could be factored out into a function?  Not worth doing on this PR, but maybe for the future.